R/dependencies.R

```diff
@@ -1695,10 +1695,10 @@ renv_dependencies_report <- function(errors) {
   enumerate(splat, function(file, problem) {
     messages <- paste("Error", problem$message, sep = ": ", collapse = "\n\n")
     text <- c(header(file), messages, "")
-    writef(text)
+    caution(text)
```

Comment:
Is it worth turning this into a `notice()`? It does have a preamble, postamble, and basically a bulleted interior.